chromium_src/components/search_engines/util.cc

```diff
@@ -27,13 +29,12 @@ void GetSearchProvidersUsingKeywordResult(
   if (template_urls && !template_urls->empty()) {
     std::vector<std::unique_ptr<TemplateURLData>> prepopulated_urls =
         TemplateURLPrepopulateData::GetPrepopulatedEngines(prefs, nullptr);
-    for (auto crit = prepopulated_urls.crbegin();
-         crit != prepopulated_urls.crend(); ++crit) {
-      auto it = find_if(
-          begin(*template_urls), end(*template_urls),
-          [&crit](std::unique_ptr<TemplateURL>& t_url1) {
-            return (t_url1->prepopulate_id() == (*crit)->prepopulate_id);
-          });
+    for (const auto& crit : base::Reversed(prepopulated_urls)) {
+      auto it =
+          find_if(begin(*template_urls), end(*template_urls),
```

Comment:
`ranges::find_if`?

Reply:
Fixed in 6720219d61654b18ecae768debeee9b0ad1cf2cb.